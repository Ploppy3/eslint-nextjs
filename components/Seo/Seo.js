import Head from "next/head";

function Seo() {
  return (
    <Head>
      <title key="title">
        The title
      </title>
      <meta
        key="description"
        name="description"
        content="The description" />
    </Head>
  );
}

export default Seo;
