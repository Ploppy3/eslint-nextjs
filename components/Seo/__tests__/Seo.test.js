import { create } from 'react-test-renderer';
import Seo from '../Seo';

// jest.mock(
//   "next/head",
//   () =>
//     function Head({ children }) {
//       return children;
//     },
// );

jest.mock('next/head', () => {
  return {
    __esModule: true,
    default: ({ children }) => {
      return children;
    },
  };
});

describe("the component", () => {
  it('should display seo', () => {
    const render = create(
      <Seo />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});