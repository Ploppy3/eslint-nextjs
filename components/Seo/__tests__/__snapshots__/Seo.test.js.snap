// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the component should display seo 1`] = `null`;
